src/diffusers/quantizers/quanto/utils.py

```diff
@@ -0,0 +1,51 @@
+from typing import Optional
+
+import torch.nn as nn
+
+from ...utils import is_accelerate_available
+
+
+if is_accelerate_available():
+    from accelerate import init_empty_weights
+
+
+def _replace_with_quanto_layers(model, quantization_config, modules_to_not_convert: list):
+    # Quanto imports diffusers internally. These are placed here to avoid circular imports
+    from optimum.quanto import QLinear, qfloat8, qint2, qint4, qint8
```

Comment:
Quanto does support Conv layers, though. Should we consider them in this PR?

Reply:
Is there a model that benefits from quantized Conv layers? I recall that it didn't work so great for SD UNets?

Reply:
Perhaps a note could be nice if we're not adding QConv. In my experiments, I was able to quantize the VAE and obtain decent results.